### ISSUE TITLE
[Windows] Textfield retains text after clearing controller

### ISSUE THREAD

**Issue by kunit1** [flutter/flutter#61832]
This issue happens only on Windows Desktop target as far as I can see.
When entering text into text field with both text controller and focus node the old text will show up again after submitting the text and entering new text.
## Steps to Reproduce

1. Create Text field
2. Add TextEditingController to text field
3. Add FocusNode to text field (Important)
4. In the onSubmitted function clear TextEditingController
5. In the onSubmitted function requestFocus on the focusNode
6. Type "Hello" into text field and hit enter
7. Type something else in now empty looking textfield

**Expected results:** 
After step 7 it is expected that only the new text is in the text field
**Actual results:** 
After step 7 the old text shows up and new input is appended to the end of the previous text
<details>
  <summary>Logs</summary>

```
[√] Flutter (Channel master, 1.21.0-2.0.pre.66, on Microsoft Windows [Version 10.0.18363.900], locale en-CA)
    • Flutter version 1.21.0-2.0.pre.66 at D:\flutter
    • Framework revision 54c9441723 (2 days ago), 2020-07-17 18:56:12 -0400
    • Engine revision d327828632
    • Dart version 2.9.0 (build 2.9.0-21.0.dev 9dca49e71e)


[√] Android toolchain - develop for Android devices (Android SDK version 28.0.3) 
    • Android SDK at C:\Users\kusay\AppData\Local\Android\Sdk
    • Platform android-28, build-tools 28.0.3
    • ANDROID_SDK_ROOT = C:\Users\kusay\AppData\Local\Android\Sdk
    • Java binary at: C:\Program Files\Android\Android Studio\jre\bin\java
    • Java version OpenJDK Runtime Environment (build 1.8.0_152-release-1248-b01)
    • All Android licenses accepted.

[√] Chrome - develop for the web
    • Chrome at C:\Program Files (x86)\Google\Chrome\Application\chrome.exe

[√] Visual Studio - develop for Windows (Visual Studio Professional 2019 16.6.2)
    • Visual Studio at C:\Program Files (x86)\Microsoft Visual Studio\2019\Professional
    • Visual Studio Professional 2019 version 16.6.30204.135
    • Windows 10 SDK version 10.0.17763.0

[!] Android Studio (version 3.3)
    • Android Studio at C:\Program Files\Android\Android Studio
    X Flutter plugin not installed; this adds Flutter specific functionality.
    X Dart plugin not installed; this adds Dart specific functionality.
    • Java version OpenJDK Runtime Environment (build 1.8.0_152-release-1248-b01)

[√] VS Code, 64-bit edition (version 1.47.1)
    • VS Code at C:\Program Files\Microsoft VS Code
    • Flutter extension version 3.12.2

[√] Connected device (4 available)
    • Windows (desktop) • windows    • windows-x64    • Microsoft Windows [Version 10.0.18363.900]
    • Web Server (web)  • web-server • web-javascript • Flutter Tools
    • Chrome (web)      • chrome     • web-javascript • Google Chrome 83.0.4103.116
    • Edge (web)        • edge       • web-javascript • Microsoft Edge 83.0.478.64
```

```
```

```
```

</details>

**Comment by stuartmorgan** [comment]
This is probably a duplicate of https://github.com/flutter/flutter/issues/59140 since Windows and Linux share the text model class.

**Comment by stuartmorgan** [comment]
Verified via debugging; it's a duplicate.
